extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -79,6 +79,9 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       this.element.classList.add(DARK_THEME_CLASS);
     }
 
+    // Add manual tap handler class to prevent advancement on tap
+    this.element.classList.add('i-amphtml-story-handle-tap-manually');
```

Comment:
This change might require more discussions. To not block the work on positioning the quiz element, can we keep it for another PR?

Reply:
Ok, I had this in its own PR (#25768) which I closed to merge into here.

I'll remove this from this PR and reopen #25768